test/test_base.py

```diff
@@ -108,7 +106,7 @@ def start_server(self):
         )
 
         self.server_thread = threading.Thread(
-            target=self.output_reader, args=(self.server.stdout, self.log.debug)
+            target=self.output_reader, args=(self.server.stdout, print)
```

Comment:
Why bring print back from earlier in the PR? https://github.com/bitcoin-dev-project/warnet/pull/400/commits/ca382c7268b456a900ca3667bc64359f649c42ef#diff-1a293b58d544ac5f9d384afd0652dc7e9e17ae7564df252dd5a8fc39543ee0e6L99

In #384, the route I took was to create an instance method that takes one argument and does a call to logger: https://github.com/mplsgrant/warnet/blob/3e2d110d3bfbf73518a0af492832c57ae04372bd/test/test_base.py#L48

```python
    def passthrough_logger(self, message):
        self.logger.info(f"{self.pod_name} - {message}")
```

But maybe we can just leave this as is and change it in #384.

Reply:
> Why bring print back from earlier in the PR? [ca382c7#diff-1a293b58d544ac5f9d384afd0652dc7e9e17ae7564df252dd5a8fc39543ee0e6L99](https://github.com/bitcoin-dev-project/warnet/commit/ca382c7268b456a900ca3667bc64359f649c42ef#diff-1a293b58d544ac5f9d384afd0652dc7e9e17ae7564df252dd5a8fc39543ee0e6L99)

The reason is this is being logged from outside already (well, here in test_base), and we are using the `output_reader` to capture all output from stdout and log it. So if we `log`, it prepends all the log info (time data etc.). We then capture this entire line via stdout and prepend a second layer of log info in `output_reader`, so we end up with double log details.
 

> But maybe we can just leave this as is and change it in #384.

Hopefully this :P

Reply:
The passtrhough logger does feel quite nice though... it would probably be nicer to get an actual (sub) logger sharing the same logfile/stdout stream though